frontend/mockups/styles.css

```
@@ -0,0 +1,193 @@
+
+.mobile-bottom-nav {
+  display: none;
+}
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.3.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)